apps/src/templates/levelSummary/FreeResponseAIEvaluation.tsx

```diff
@@ -15,18 +16,26 @@ interface StudentResponse {
   aiEvaluation: string;
 }
 
+interface LevelData {
+  levelInstructions: string;
+  levelId: number;
+  scriptId: number;
+}
+
 interface FreeResponseAIEvaluationProps {
   responses: StudentResponse[];
-  levelInstructions: string;
+  levelData: LevelData;
 }
+
 const FreeResponseAIEvaluation: React.FunctionComponent<
   FreeResponseAIEvaluationProps
-> = ({responses, levelInstructions}) => {
+> = ({responses, levelData}) => {
   const [evaluationsPending, setEvaluationsPending] = useState<boolean>(false);
   const [evaluations, setEvaluations] = useState<StudentResponse[]>([]);
   const [evaluationCount, setEvaluationCount] = useState<number>(0);
   const [aiSummary, setAiSummary] = useState<string>('');
-  const evaluationComplete = responses.length === evaluationCount;
+  const evaluationComplete =
+    evaluationCount > 0 && responses.length === evaluationCount;
```

Comment:
I updated the logic here because in the case of no student responses to a question, `evaluationComplete` would resolve to `true` which would then needlessly kick off a call to the `summarizeStudentEvaluations` function.